apps/earn-protocol-institutions/components/layout/VaultsDropdownWrapper/VaultsDropdownWrapper.module.css

```
@@ -4,4 +4,10 @@
   .dropdownSelected {
     width: 100%;
   }
-}
\ No newline at end of file
+
+  .contentWrapper {
+    min-width: unset;
+    width: 100%;
+    min-width: 200px;
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

min-width is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)